module/PowerNSX.psm1

```diff
@@ -9111,12 +9111,14 @@ function Get-NsxLogicalSwitch {
     Transport Zone.
 
     .EXAMPLE
+    Get-NsxTransportZone | Get-NsxLogicalswitch -name LS1
 
-    Example1: Get a named Logical Switch
-    PS C:\> Get-NsxTransportZone | Get-NsxLogicalswitch -name LS1
+    Get a named Logical Switch
 
-    Example2: Get all logical switches in a given transport zone.
-    PS C:\> Get-NsxTransportZone | Get-NsxLogicalswitch
+    .EXAMPLE
+    Get-NsxTransportZone | Get-NsxLogicalswitch
+
+    Get all logical switches in a given transport zone.
```

Comment:
either amend the Get-TZ to specify a TZ name, or update the comment to be 'in all transport zones'